src/Bab/RabbitMq/VhostManager.php

```diff
@@ -18,10 +18,7 @@ class VhostManager
     public function __construct(array $credentials, Action $action, HttpClient $httpClient)
     {
         $this->credentials = $credentials;
-        if ('/' === $this->credentials['vhost']) {
-            $this->credentials['vhost'] = '%2f';
-        }
-
+        $this->credentials['vhost'] = str_replace('/', '%2F', $this->credentials['vhost']);
```

Comment:
Is there a reason to use `%2F` instead of `%2f`?

Reply:
... no. Do you prefer lowercase ?

Reply:
As this is what is used is the [official documentation](https://www.rabbitmq.com/uri-spec.html) yes please. :)